python/torch_mlir/extras/fx_importer.py

```diff
@@ -273,6 +282,11 @@
 # so split by + and 0 index will always give the base version
 _IS_TORCH_2_1_OR_EARLIER = torch.__version__.split("+")[0] <= "2.1.0"
 
+# The value here is chosen to match that of sys.maxint (or sys.maxsize in case
+# of Python3.x). This is to maintain compatibility with older version of PyTorch.
+MLIR_INT_INFINITY = (2**63 - 1) - 1
```

Comment:
To avoid errors like these, why not use `INT64_MAX` and `INT64_MIN` from LLVM/stdint.h directly?
https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/llvm/llvm-project%24+INT64_MAX&patternType=keyword&sm=0